cmd/oras/internal/display/status/interface.go

```diff
@@ -66,6 +66,13 @@ type CopyHandler interface {
 	StopTracking() error
 }
 
+// ManifestPushHandler handles status output for manifest push command.
+type ManifestPushHandler interface {
+	OnPushSkipped() error
```

Comment:
Better rename to below so it does not conflict with other commands' handlers
```suggestion
	OnManifestPushSkipped() error
```